<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>874</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>无标题</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QMdiArea" name="mdi"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>874</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="act_New"/>
    <addaction name="act_New_Window"/>
    <addaction name="act_Open"/>
    <addaction name="act_Save"/>
    <addaction name="act_SaveAs"/>
    <addaction name="act_CloseAll"/>
    <addaction name="separator"/>
    <addaction name="act_PageSetup"/>
    <addaction name="act_Printing"/>
    <addaction name="separator"/>
    <addaction name="act_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="act_Redo"/>
    <addaction name="act_Undo"/>
    <addaction name="separator"/>
    <addaction name="act_Cut"/>
    <addaction name="act_Copy"/>
    <addaction name="act_Paste"/>
    <addaction name="act_Delete"/>
    <addaction name="separator"/>
    <addaction name="act_Bing"/>
    <addaction name="act_Find"/>
    <addaction name="act_FindNext"/>
    <addaction name="act_FindPrevious"/>
    <addaction name="act_Replace"/>
    <addaction name="act_Goto"/>
    <addaction name="separator"/>
    <addaction name="act_SelectAll"/>
    <addaction name="act_TimeDate"/>
   </widget>
   <widget class="QMenu" name="menu_O">
    <property name="title">
     <string>格式(&amp;O)</string>
    </property>
    <addaction name="act_Wrap"/>
    <addaction name="act_Font"/>
   </widget>
   <widget class="QMenu" name="menu_V">
    <property name="title">
     <string>查看(&amp;V)</string>
    </property>
    <widget class="QMenu" name="menus">
     <property name="title">
      <string>缩放</string>
     </property>
     <addaction name="act_ZoomIn"/>
     <addaction name="act_ZoomOut"/>
     <addaction name="act_DefaultZoom"/>
    </widget>
    <addaction name="menus"/>
    <addaction name="act_StatusBar"/>
    <addaction name="separator"/>
    <addaction name="act_Mdi"/>
    <addaction name="act_LevelExpansion"/>
    <addaction name="act_FlatOut"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="act_ViewHelp"/>
    <addaction name="act_Feedback"/>
    <addaction name="separator"/>
    <addaction name="act_About"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_E"/>
   <addaction name="menu_O"/>
   <addaction name="menu_V"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QStatusBar" name="StatusB"/>
  <action name="act_New">
   <property name="text">
    <string>新建</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="act_New_Window">
   <property name="text">
    <string>新窗口</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+N</string>
   </property>
  </action>
  <action name="act_Open">
   <property name="text">
    <string>打开...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="act_Save">
   <property name="text">
    <string>保存</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="act_SaveAs">
   <property name="text">
    <string>另存为...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="act_PageSetup">
   <property name="text">
    <string>页面设置...</string>
   </property>
  </action>
  <action name="act_Printing">
   <property name="text">
    <string>打印...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="act_Exit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="act_Undo">
   <property name="text">
    <string>撤销</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="act_Cut">
   <property name="text">
    <string>剪切</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="act_Copy">
   <property name="text">
    <string>复制</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="act_Paste">
   <property name="text">
    <string>粘贴</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="act_Delete">
   <property name="text">
    <string>删除</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="act_Bing">
   <property name="text">
    <string>使用Bing搜索...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="act_Find">
   <property name="text">
    <string>查找...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="act_FindNext">
   <property name="text">
    <string>查找下一个</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="act_FindPrevious">
   <property name="text">
    <string>查找上一个</string>
   </property>
   <property name="shortcut">
    <string>Shift+F3</string>
   </property>
  </action>
  <action name="act_Replace">
   <property name="text">
    <string>替换</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="act_Goto">
   <property name="text">
    <string>转到</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="act_SelectAll">
   <property name="text">
    <string>全选</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="act_TimeDate">
   <property name="text">
    <string>时间/日期</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="act_Wrap">
   <property name="text">
    <string>自动换行</string>
   </property>
  </action>
  <action name="act_Font">
   <property name="text">
    <string>字体...</string>
   </property>
  </action>
  <action name="act_ZoomIn">
   <property name="text">
    <string>放大</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+=</string>
   </property>
  </action>
  <action name="act_ZoomOut">
   <property name="text">
    <string>缩小</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="act_DefaultZoom">
   <property name="text">
    <string>恢复默认缩放</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="act_StatusBar">
   <property name="text">
    <string>状态栏</string>
   </property>
  </action>
  <action name="act_ViewHelp">
   <property name="text">
    <string>查看帮助</string>
   </property>
  </action>
  <action name="act_Feedback">
   <property name="text">
    <string>发送反馈</string>
   </property>
  </action>
  <action name="act_About">
   <property name="text">
    <string>关于记事本</string>
   </property>
  </action>
  <action name="act_Redo">
   <property name="text">
    <string>重做</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="act_Mdi">
   <property name="text">
    <string>MDI模式</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="act_LevelExpansion">
   <property name="text">
    <string>层级展开</string>
   </property>
  </action>
  <action name="act_FlatOut">
   <property name="text">
    <string>平铺展开</string>
   </property>
  </action>
  <action name="act_CloseAll">
   <property name="text">
    <string>关闭全部</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>act_Exit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
